tf2_ros_py/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Is there any reason we can't make this a pure python package (no CMakeLists.txt at all)?  I think that would be more pythonic.